config/config.go

```diff
@@ -530,6 +535,19 @@
 		return err
 	}
 
+	log.WithField("filepath", _config.System.User.PasswdFile).Debug("ensuring passwd file exists")
+	if passwd, err := os.Create(_config.System.User.PasswdFile); err != nil {
```

Comment:
What are your thoughts on this finding @parkervcp?  It doesn't seem terrible to me since wings already runs as root, so it should be assumed anyone with access to the config file can do nefarious things.

Worst case(?) when file permissions are too permissible on the wings config file it could allow a non-root attacker to replace the passwd file for all containers on a node with their own.  We could potentially mitigate this by doing a permission check on the wings config file and refuse to start if the config file has broad write permissions.

Reply:
This isn't even configurable on the panel side.

In theory the only people able to to modify the file path are those with access to the wings config on the host. People who would probably already have root access.

Reply:
Sounds good to me, just wanted to make sure we were on the same page for the implications of ignoring this.